auto_py_to_exe/utils.py

```diff
@@ -39,10 +40,37 @@ def open_output_in_explorer(output_directory, input_filename, is_one_file):
         return False
     return True
 
+def check_latest_pyinstaller_version():
+    """Check the latest version of PyInstaller"""
+    try:
+        response = requests.get("https://api.github.com/repos/pyinstaller/pyinstaller/releases/latest")
+        response.raise_for_status()
+        latest_version = response.json()["tag_name"]
+        if latest_version.startswith("v"):
+            latest_version = latest_version[1:]
+        return latest_version
+    except requests.exceptions.RequestException:
+        return None
 
 def get_warnings():
     warnings = []
 
+    # Check pyinstaller version is it latest
+    try:
+        latest_version = check_latest_pyinstaller_version()
```

Comment:
If we add more version checks, good to scope the name to "pyinstaller"

```suggestion
        latest_pyinstaller_version = get_latest_pyinstaller_version()
```